src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.RuntimeConfig.targets

```diff
@@ -51,6 +51,16 @@ See: https://github.com/dotnet/runtime/blob/b13715b6984889a709ba29ea8a1961db469f
         Value="$([MSBuild]::ValueOrDefault('$(_AndroidUseManagedTypeMap)', 'false'))"
         Trim="true"
     />
+    <RuntimeHostConfigurationOption Include="Microsoft.Android.Runtime.RuntimeFeature.IsMonoRuntime"
+        Condition=" '$(_AndroidRuntime)' == 'MonoVM' "
```

Comment:
Should we also have a `false` case with the inverse condition so that the feature switch is always explicitly set to a value?

Reply:
This one is just setting the default values anyway.

We have a `*.CoreClr.targets` and `*.NativeAot.targets` that are only imported on each specific runtime. I can create a `*.Mono.targets` as well, and we'd have a lot less `$(_AndroidRuntime)` checks.